src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/compute/sdk/AzureVirtualMachineOperationSDKTest.java

```diff
@@ -397,9 +400,10 @@ public void testBuildAzureVirtualMachineObservable() throws InvalidParameterExce
 
         PowerMockito.mockStatic(AzureNetworkSDK.class);
         NetworkInterface networkInterfaceExcepted = Mockito.mock(NetworkInterface.class);
+        Optional<NetworkInterface> networkInterfaceExceptedOptional = Optional.ofNullable(networkInterfaceExcepted);
```

Comment:
I think you misspelled "expected"

Reply:
Done.
I've made the change in the branch azure-compute-plugin-four-pr-fixes